cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -312,13 +339,41 @@ protected ApplicationManager deployApplication(Id.Namespace namespace,
   }
 
   protected ApplicationManager deployApplication(Class<? extends Application> applicationClz) throws IOException {
-    return deployApplication(Id.Namespace.DEFAULT, applicationClz);
```

Comment:
o actually nvm. this is probably the only method that defaults to default namespace